projects/storefrontlib/src/cms-structure/routing/default-routing-config.ts

```diff
@@ -40,6 +40,12 @@ export const defaultStorefrontRoutesConfig: RoutesConfig = {
   orders: {
     paths: ['my-account/orders'],
   },
+
+  // configuration related routes
+  configureCPQCONFIGURATOR: {
```

Comment:
1. Just FYI: this key is only the semantic name of the route (to use in `routing.go` and `| cxUrl` pipe), so it doesn't have to be the same as the actual (part of) url. 
2. I'm wondering if we can make it a little more readable and even more generic (not to introduce separate semantic routes for each configurator, but drive it by the params). What do you think about:
```typescript
configure: {
   paths: ['configure/:configurator/:rootProduct']
}
```
Then in the CMS the page labels for different configurators could be:
/configure/CPQCONFIGURATOR
/configure/sth-else-configurator
etc.

Reply:
Yes. Definitively a nicer approach. Reason for my current approach simply was the _existing_ accelerator content. But from the conversation we had this morning I assume that we can change the content slightly if we get an easier, generic and more understandable behaviour.
Will CMS and SPA allow for labels like /configure/TEXTFIELD?
Anyhow I will try this

Reply:
We need this semantic key configureCPQCONFIGURATOR because we are composing it dynamically from the backend configurator type...

Reply:
For a route 'configure/:configurator/:rootProduct': This unfortunately does not work, as the pages are not found in CMS when requesting e.g. configure/CPQCONFIGURATOR. On the other hand, we do need distinct pages per configurator, for page composition => From my point of view we need to stick to specific routes per configurator.

Reply:
> the pages are not found in CMS when requesting e.g. configure/CPQCONFIGURATOR
If we change the pagel labels in the CMS from `configuratorCPQCONFIGURATOR` to `/configurator/CPQCONFIGURATOR`, won't it fly (similar for other configurators)?

Agreed that the drawback of changing the CMS data is harder integration for existing Customers. So we would need enough good reason to change it.

Reply:
Unfortunately this does not fly, I tried it yesterday. Agree that it would have been nicer..